Curtains/Curtains.Application/ConstructorObjects/BedspreadsProduct.cs

```diff
@@ -0,0 +1,17 @@
+using Curtains.Application.ConstructorObjects.Interfaces;
+
+namespace Curtains.Application.ConstructorObjects
+{
+    public class BedspreadsProduct : IProduct
+    {
+        /// <summary>
+        ///  This property represents size of pillows.
```

Comment:
Size of Pillows?
Why property name Title, but it represent size?

Reply:
rewrited